docs/source/04_user_guide/08_advanced_io.md

```diff
@@ -73,6 +73,10 @@ class MyOwnDataSet(AbstractVersionedDataSet):
         save_path = self._get_save_path()
         df.to_csv(save_path)
 
+    def _exists(self) -> bool:
+        path = self._get_load_path()
+        return path.is_file()
```

Comment:
Well spotted! I just noticed the `super()` call is also out of date, it should be passing a purepath, e.g. `PurePosixPath(filepath)`. Do you mind making that change here on line 64, and in `kedro/io/core.py` - line 479 or thereabout? 

Personal preference, I would also leave filepath and version as the first args, then followed by `param1` and an optional `param2`.

Reply:
💯 Agree, I like to keep arguments in the natural order they get called.  Gives me goosebumps that someone is going to not use keyword arguments and call it backwards.